manifests/install.pp

```diff
@@ -8,8 +8,10 @@
   include 'archive'
 
   file { $solr::staging_dir:
-    ensure  => directory,
-    recurse => true,
+    ensure    => directory,
+    recurse   => true,
+    # Suppress "Warning: The directory '/opt/staging' contains #### entries, which exceeds the default soft limit 1000"
+    max_files => 3000,
```

Comment:
I'd rather set this to `-1` and use a more comprehensive comment. Could you update your PR please? :)